packages/contracts-bedrock/slither.config.json

```diff
@@ -1,9 +1,11 @@
 {
-  "detectors_to_exclude": "assembly-usage,block-timestamp,naming-convention,solc-version,low-level-calls,boolean-equality",
```

Comment:
This list on the right was growing by a 'whack-a-mole' like process.
The remaining list is deliberate.